app/settings/page.tsx

```diff
@@ -0,0 +1,77 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import { themeChange } from "theme-change";
+
+import PageTitle from "../../components/ui/PageTitle";
+
+const Settings = () => {
+  const themeValues = [
+    "light",
+    "cupcake",
+    "bumblebee",
+    "emerald",
+    "corporate",
+    "synthwave",
+    "retro",
+    "cyberpunk",
+    "valentine",
+    "halloween",
+    "garden",
+    "forest",
+    "aqua",
+    "lofi",
+    "pastel",
+    "fantasy",
+    "wireframe",
+    "black",
+    "luxury",
+    "dracula",
+    "cmyk",
+    "autumn",
+    "business",
+    "acid",
+    "lemonade",
+    "night",
+    "coffee",
+    "winter",
+  ];
+
+  const [theme, setTheme] = useState<string>();
```

Comment:
```suggestion
  const [theme, setTheme] = useState<typeof THEME_VALUES>();
```

This will make it more rigid (so you can't just put any string in)